test/spec/vis/layers-factory.spec.js

```diff
@@ -106,20 +109,22 @@ describe('vis/layers-factory', function () {
           spyOn(LayersFactory, 'isHttps').and.returnValue(true);
         });
 
-        it("should not convert '" + httpsUrlTemplate + "'", function () {
-          var layerModel = layersFactory.createLayer('tiled', {
-            urlTemplate: httpsUrlTemplate
+        describe('and is high resolution screen', function () {
```

Comment:
iwhere is the high resolution screen set?

Reply:
I am going to remove this test since it is not longer necessary.